adapters/handlers/graphql/local/local_component_test.go

```diff
@@ -54,15 +54,14 @@ func Test_GraphQLNetworkBuild(t *testing.T) {
 						},
 					},
 				},
-				Things: &models.Schema{},
 			},
 		},
 
 		// This tests asserts that a things-only schema doesn't lead to errors.
```

Comment:
I think this test is redundant, as it effectively tests the same thing as the one before :)